R/pull.R

```diff
@@ -65,22 +71,22 @@ maybe_repair <- function(x) {
 }
 
 
-pull_metrics <- function(resamples, res, control) {
```

Comment:
The remainder of the changes in this file just newly pass `order` along to `pulley()`.